frontend/src/cards/UnknownsMapCard.tsx

```diff
@@ -194,15 +190,6 @@ function UnknownsMapCardWithKey(props: UnknownsMapCardProps) {
 
         return (
           <>
-            <CardContent className={styles.SmallMarginContent}>
-              <MapBreadcrumbs
-                fips={props.fips}
-                updateFipsCallback={props.updateFipsCallback}
-                scrollToHashId="unknown-demographic-map"
-              />
-            </CardContent>
-            <Divider />
-
```

Comment:
can you make an issue do look at re-introducing the breadcrumbs somewhere on the card in a future PR?